core/src/test/scala/org/apache/spark/deploy/SparkSubmitSuite.scala

```diff
@@ -575,8 +575,13 @@ class SparkSubmitSuite
   // NOTE: This is an expensive operation in terms of time (10 seconds+). Use sparingly.
   private def runSparkSubmit(args: Seq[String]): Unit = {
     val sparkHome = sys.props.getOrElse("spark.test.home", fail("spark.test.home is not set!"))
+    val sparkSubmit = if (Utils.isWindows) {
+      Seq(new File("..\\bin\\spark-submit.cmd").getAbsolutePath) ++ args
```

Comment:
Yes, I've tried it. To be honest, I have no idea why there is such difference between win and Linux. `new File(".\\bin\\spark-submit").getAbsolutePath` here return me the path under `core`. I've tried `mvn` only though, `sbt` doesn't work well on windows.